test_unstructured/partition/epub/test_epub.py

```diff
@@ -222,4 +218,4 @@ def test_partition_epub_respects_detect_language_per_element():
     filename = "example-docs/language-docs/eng_spa_mult.epub"
     elements = partition_epub(filename=filename, detect_language_per_element=True)
     langs = [element.metadata.languages for element in elements]
-    assert langs == [["eng"], ["eng"], ["spa", "eng"], ["eng"], ["eng"], ["spa"]]
+    assert langs == [["eng"], ["spa", "eng"], ["eng"], ["eng"], ["spa"]]
```

Comment:
Do you know why there's one fewer elements created for this document?

Reply:
I haven't looked closely, but if I had to guess maybe a section change happened in the middle of one of the current elements.

Reply:
That would make sense, but I will take a look to confirm.

Reply:
@qued for some reason I'm having trouble checking out to this branch locally, so can you check?